src/routes/RequestsRoute.js

```diff
@@ -540,10 +545,14 @@ class RequestsRoute extends React.Component {
 
   // Called as a map function
   addRequestFields(r) {
+    const { requestLevel } = r;
+
     return Promise.all(
       [
         this.findResource('user', r.requesterId),
-        this.findResource('requestsForItem', r.itemId),
+        requestLevel === REQUEST_LEVEL_TYPES.ITEM
```

Comment:
Could you please double-check should we show requests for `Instance`, if we working with `Item` level request

Reply:
done